.github/workflows/todoapp_pipeline.yaml

```diff
@@ -34,12 +50,26 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_HUB_ACCESS_USER }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Login to azure
+        uses: azure/login@v1
+        with:
+          creds: '${{ secrets.AZURE_CREDENTIALS }}'
+      - name: 'Run Azure CLI commands'
+        run: |
+          az account show
+          az group list
+          pwd        
```

Comment:
You should find that this isn't necessary if you've got the ARM_CLIENT_ID etc. variables set correctly, but equally no particular harm with it